src/org/labkey/test/components/ui/grids/CustomizeGridDialog.java

```diff
@@ -0,0 +1,541 @@
+package org.labkey.test.components.ui.grids;
+
+import org.junit.Assert;
+import org.labkey.test.BootstrapLocators;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.UpdatingComponent;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.html.Checkbox;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Wraps EntityFieldFilterModal.tsx in UI components.
+ */
+public class CustomizeGridDialog extends ModalDialog
```

Comment:
Wrong product component (Copy/paste error I presume). Class name should match also.
```suggestion
/**
 * Wraps CustomizeGridViewModal.tsx in UI components.
 */
public class CustomizeGridViewDialog extends ModalDialog
```